### PR TITLE
[K3.0.7] Menu item Profile > Use Integration = No: Click in Kunena profile on edit gives a redirect to the CP Profile #1835

### PR DESCRIPTION
**Pull request by 810**
#1835

### CODE REVIEW

**Comment by 810** [comment]
@rich20 could you confirm this one

**Comment by rich20** [comment]
- [x] Success

File: components/com_kunena/site/views/user/view.html.php [hunk 1]
```diff
@@ -89,8 +89,8 @@ protected function displayCommon($tpl = null) {
 			$this->profile->save();
 		}
 		if ($this->profile->userid == $this->me->userid) {
-			if ($this->do != 'edit') $this->editlink = $this->profile->getLink ( JText::_('COM_KUNENA_EDIT').' &raquo;', JText::_('COM_KUNENA_EDIT').' &raquo;', 'nofollow', 'edit' );
-			else $this->editlink = $this->profile->getLink ( JText::_('COM_KUNENA_BACK').' &raquo;', JText::_('COM_KUNENA_BACK').' &raquo;', 'nofollow' );
+			if ($this->do != 'edit') $this->editlink = '<a href="index.php?option=com_kunena&view=user&layout=edit" rel="nofollow">' . JText::_('COM_KUNENA_EDIT') . '</a>';
+			else $this->editlink = '<a href="index.php?option=com_kunena&view=user&userid=' . $this->me->userid . '" rel="nofollow">' . JText::_('COM_KUNENA_BACK') . '</a>';
```
Comment (mahagr) [inline components/com_kunena/site/views/user/view.html.php]: Removing Joomla routing isn't an solution and totally breaks our internal routing mechanism.

**Comment by mahagr** [comment]
We need better solution for this one. Reverted

**Pull request merged**